shell/pom.xml

```diff
@@ -98,5 +98,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <!-- version needs to be >= 3.4.0 -->
+      <version>3.5.13</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
You can put this in the project root pom, since other child projects will depend on this soon. Also, please make sure this artifact has the same version as the `mockito-core`.

Reply:
+1above, I'm also looking forward to using this new feature in other projects

Reply:
I have added it into `    <!-- Dependencies in the test scope. -->`